src/launcher/java/org/truffleruby/launcher/options/CommandLineOptions.java

```diff
@@ -54,6 +53,10 @@
     private boolean forceStdin;
     private boolean shouldPrintShortUsage;
 
+    // TODO (pitr-ch 26-Jul-2017): Move to Options when implementing -s option
+    // Currently not used
```

Comment:
I would just re-add it manually, it's literally a field declaration and a getter, right?